### ISSUE TITLE
ERROR: Unable to extract jwplatform id;

### ISSUE THREAD

**Issue by cowcrash** [rg3/youtube-dl#27526]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2020.12.22**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
PASTE VERBOSE LOG HERE
```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', '-k', '--ignore-config', '--verbose', 'https://www.tele5.de/rellik/ganze-folge/episode-1/']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2020.12.22
[debug] Python version 3.7.3 (CPython) - Linux-4.19.0-13-amd64-x86_64-with-debian-10.7
[debug] exe versions: ffmpeg 4.1.6, ffprobe 4.1.6, rtmpdump 2.4
[debug] Proxy map: {}
[debug] Using fake IP 53.33.143.108 (DE) as X-Forwarded-For.
[Tele5] episode-1: Downloading webpage
ERROR: Unable to extract jwplatform id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/youtube_dl/YoutubeDL.py", line 803, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/youtube_dl/YoutubeDL.py", line 824, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3/dist-packages/youtube_dl/extractor/common.py", line 532, in extract
    ie_result = self._real_extract(url)
  File "/usr/lib/python3/dist-packages/youtube_dl/extractor/tele5.py", line 102, in _real_extract
    jwplatform_id = extract_id(JWPLATFORM_ID_RE, 'jwplatform id')
  File "/usr/lib/python3/dist-packages/youtube_dl/extractor/tele5.py", line 95, in extract_id
    default=default)
  File "/usr/lib/python3/dist-packages/youtube_dl/extractor/common.py", line 1019, in _html_search_regex
    res = self._search_regex(pattern, string, name, default, fatal, flags, group)
  File "/usr/lib/python3/dist-packages/youtube_dl/extractor/common.py", line 1010, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract jwplatform id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

## Description

WRITE DESCRIPTION HERE
I want do download that video. Other downloads from 'tele5.de' succeded 8(

**Comment by SebiderSushi** [comment]
tele5 seems to have started to DRM protect most content very recently. Maybe `youtube-dl` should improve DRM detection here.

On Firefox you can check if DRM is being used on a page by looking at your address bar, there should be a clickable indicator while DRM is in use:
![photo_2021-01-26_04-48-47](https://user-images.githubusercontent.com/23618858/105798167-d1d18d00-5f91-11eb-9f45-e178b10fa17f.jpg)

If you click "Configure..." you can turn off DRM use in Firefox. You can also manually type `about:preferences` in your Address bar and scroll down until you see the DRM option (or search for "DRM") and disable it.
If you try to play DRM protected content with that option disabled, Firefox will display a banner notifying you about it:

![image_2021-01-26_04-28-34](https://user-images.githubusercontent.com/23618858/105798336-3ee52280-5f92-11eb-94fc-25ac9f52375c.png)

This Banner is also displayed if i try to access the reported URL at `https://www.tele5.de/rellik/ganze-folge/episode-1/` (though i get redirected to `https://www.tele5.de/rellik/ganze-folge/episode-4/` as episode 1 is no longer online).

**Comment by SebiderSushi** [comment]
Strangely, it turns out that even though i get that banner on the show of OPs error URL, `youtube-dl` is currently able to download it as opposed to when OP tried it, producing an unencrypted and playable file:

```
[sebi@manjaro ~]$ youtube-dl --verbose https://www.tele5.de/rellik/ganze-folge/episode-4/ 
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', 'https://www.tele5.de/rellik/ganze-folge/episode-4/']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.01.24.1
[debug] Python version 3.9.1 (CPython) - Linux-5.10.7-3-MANJARO-x86_64-with-glibc2.32
[debug] exe versions: ffmpeg 4.3.1, ffprobe 4.3.1, rtmpdump 2.4
[debug] Proxy map: {}
[debug] Using fake IP 53.44.18.113 (DE) as X-Forwarded-For.
[Tele5] episode-4: Downloading webpage
[debug] Using fake IP 53.200.24.107 (DE) as X-Forwarded-For.
[JWPlatform] ZDYjqCyN: Downloading JSON metadata
[JWPlatform] ZDYjqCyN: Downloading m3u8 information
[debug] Default format spec: bestvideo+bestaudio/best
[debug] Invoking downloader on 'https://tele5.akamaized.net/B4003823_b2c13093209cb41f254fa0befaea7a51/media-3/stream.m3u8'
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 519
[download] Destination: Episode 4-ZDYjqCyN.mp4
[download]   0.2% of ~781.82MiB at 459.94KiB/s ETA 30:40^C
ERROR: Interrupted by user
```


However, this example fails: 
```
[sebi@manjaro ~]$ youtube-dl --verbose https://www.tele5.de/filme/apocalypse-now/
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', 'https://www.tele5.de/filme/apocalypse-now/']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.01.24.1
[debug] Python version 3.9.1 (CPython) - Linux-5.10.7-3-MANJARO-x86_64-with-glibc2.32
[debug] exe versions: ffmpeg 4.3.1, ffprobe 4.3.1, rtmpdump 2.4
[debug] Proxy map: {}
[debug] Using fake IP 53.106.137.162 (DE) as X-Forwarded-For.
[Tele5] apocalypse-now: Downloading webpage
[debug] Using fake IP 53.158.163.168 (DE) as X-Forwarded-For.
[JWPlatform] pXcY1AOD: Downloading JSON metadata
[JWPlatform] pXcY1AOD: Downloading m3u8 information
[debug] Default format spec: bestvideo+bestaudio/best
[debug] Invoking downloader on 'https://tele5.akamaized.net/A4004983_0b472737de7574b5d8901cd69e06f155/hls/media-3/stream.m3u8'
[hlsnative] Downloading m3u8 manifest
WARNING: hlsnative has detected features it does not support, extraction will be delegated to ffmpeg
[download] Destination: Apocalypse Now-pXcY1AOD.mp4
[debug] ffmpeg command line: ffmpeg -y -loglevel verbose -headers 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.41 Safari/537.36
Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Encoding: gzip, deflate
Accept-Language: en-us,en;q=0.5
X-Forwarded-For: 53.158.163.168
' -i https://tele5.akamaized.net/A4004983_0b472737de7574b5d8901cd69e06f155/hls/media-3/stream.m3u8 -c copy -f mp4 'file:Apocalypse Now-pXcY1AOD.mp4.part'
ffmpeg version n4.3.1 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 10.2.0 (GCC)
  configuration: --prefix=/usr --disable-debug --disable-static --disable-stripping --enable-amf --enable-avisynth --enable-cuda-llvm --enable-lto --enable-fontconfig --enable-gmp --enable-gnutls --enable-gpl --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libdav1d --enable-libdrm --enable-libfreetype --enable-libfribidi --enable-libgsm --enable-libiec61883 --enable-libjack --enable-libmfx --enable-libmodplug --enable-libmp3lame --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libopenjpeg --enable-libopus --enable-libpulse --enable-librav1e --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-nvdec --enable-nvenc --enable-shared --enable-version3
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
  libpostproc    55.  7.100 / 55.  7.100
[tcp @ 0x55c804a2db80] Starting connection attempt to 23.55.161.33 port 443
[tcp @ 0x55c804a2db80] Successfully connected to 23.55.161.33 port 443
[hls @ 0x55c804a29940] Skip ('#EXT-X-VERSION:5')
[hls @ 0x55c804a29940] Skip ('#EXT-X-INDEPENDENT-SEGMENTS')
[AVIOContext @ 0x55c804aca740] Statistics: 32768 bytes read, 0 seeks
https://tele5.akamaized.net/A4004983_0b472737de7574b5d8901cd69e06f155/hls/media-3/stream.m3u8: Invalid data found when processing input


ERROR: ffmpeg exited with code 1
  File "/home/sebi/.local/bin/youtube-dl", line 8, in <module>
    sys.exit(main())
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/__init__.py", line 475, in main
    _real_main(argv)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/__init__.py", line 465, in _real_main
    retcode = ydl.download(all_urls)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 2055, in download
    res = self.extract_info(
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 799, in extract_info
    return self.__extract_info(url, ie, download, extra_info, process)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 806, in wrapper
    return func(self, *args, **kwargs)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 838, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 876, in process_ie_result
    return self.extract_info(ie_result['url'],
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 799, in extract_info
    return self.__extract_info(url, ie, download, extra_info, process)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 806, in wrapper
    return func(self, *args, **kwargs)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 838, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 872, in process_ie_result
    return self.process_video_result(ie_result, download=download)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 1679, in process_video_result
    self.process_info(new_info)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 1963, in process_info
    success = dl(filename, info_dict)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 1902, in dl
    return fd.download(name, info)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/downloader/common.py", line 366, in download
    return self.real_download(filename, info_dict)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/downloader/hls.py", line 78, in real_download
    return fd.real_download(filename, info_dict)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/downloader/external.py", line 63, in real_download
    self.report_error('%s exited with code %d' % (
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/downloader/common.py", line 165, in report_error
    self.ydl.report_error(*args, **kargs)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 628, in report_error
    self.trouble(error_message, tb)
  File "/home/sebi/.local/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 590, in trouble
    tb_data = traceback.format_list(traceback.extract_stack())
```


This is the head of the failing manifest:
```
$ curl https://tele5.akamaized.net/A4004983_0b472737de7574b5d8901cd69e06f155/hls/media-3/stream.m3u8
#EXTM3U
#EXT-X-VERSION:5
#EXT-X-PLAYLIST-TYPE:VOD
#EXT-X-INDEPENDENT-SEGMENTS
#EXT-X-TARGETDURATION:6
#EXT-X-MEDIA-SEQUENCE:0
#EXT-X-KEY:METHOD=SAMPLE-AES,URI="skd://78161b4742c6455aab940736aac8ecc7",KEYFORMAT="com.apple.streamingkeydelivery",KEYFORMATVERSIONS="1"
#EXTINF:6.720000,
segment-0.ts
#EXTINF:6.720000,
segment-1.ts
#EXTINF:6.720000,
segment-2.ts
```

On a windows machine that i have no access to someone else observed more precise FFmpeg error messages, they only sent me a screenshot but the error messages were basically repeated errors of
```
Error opening fragment x
SAMPLE-AES encryption is not supported yet
```

**Comment by SebiderSushi** [comment]
So looks to me like tele5.de still offers a DRM-free fallback for select media that `youtube-dl` uses.
For `https://www.tele5.de/rellik/ganze-folge/episode-4/ ` these are the manifests that i am getting:

`youtube-dl`:
```
$ curl $(youtube-dl -g https://www.tele5.de/rellik/ganze-folge/episode-4/)
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-PLAYLIST-TYPE:VOD
#EXT-X-INDEPENDENT-SEGMENTS
#EXT-X-TARGETDURATION:6
#EXT-X-MEDIA-SEQUENCE:0
#EXTINF:6.720000,
segment-0.ts
#EXTINF:6.720000,
segment-1.ts
#EXTINF:6.720000,
segment-2.ts
#EXTINF:6.720000,
segment-3.ts
#EXTINF:6.720000,
segment-4.ts
#EXTINF:6.720000,
segment-5.ts
#EXTINF:6.720000,
segment-6.ts
#EXTINF:6.720000,
segment-7.ts
#EXTINF:6.720000,
segment-8.ts
#EXTINF:6.720000,
segment-9.ts
[...]
```


Firefox (manifest URL acquired from Developer tools network tab while watching with DRM content enabled):
```
$ curl https://tele5.akamaized.net/B4003823_b2c13093209cb41f254fa0befaea7a51/dash/manifest.mpd
<?xml version="1.0" ?>
<MPD mediaPresentationDuration="PT58M3.880S" minBufferTime="PT10.00S" profiles="urn:mpeg:dash:profile:isoff-live:2011" type="static" xmlns="urn:mpeg:dash:schema:mpd:2011" xmlns:cenc="urn:mpeg:cenc:2013" xmlns:mspr="urn:microsoft:playready">
  <!-- Created with Bento4 mp4-dash.py, VERSION=2.0.0-637 -->
  <Period>
    <!-- Video -->
    <AdaptationSet maxHeight="576" maxWidth="1024" mimeType="video/mp4" segmentAlignment="true" startWithSAP="1">
      <!-- MPEG Common Encryption -->
      <ContentProtection cenc:default_KID="a069ed8d-c795-43bf-9239-84dbd8b3878a" schemeIdUri="urn:mpeg:dash:mp4protection:2011" value="cenc"/>
      <!-- PlayReady -->
      <ContentProtection schemeIdUri="urn:uuid:9a04f079-9840-4286-ab92-e65be0885f95" value="2.0">
        <mspr:pro>jAIAAAEAAQCCAjwAVwBSAE0ASABFAEEARABFAFIAIAB4AG0AbABuAHMAPQAiAGgAdAB0AHAAOgAvAC8AcwBjAGgAZQBtAGEAcwAuAG0AaQBjAHIAbwBzAG8AZgB0AC4AYwBvAG0ALwBEAFIATQAvADIAMAAwADcALwAwADMALwBQAGwAYQB5AFIAZQBhAGQAeQBIAGUAYQBkAGUAcgAiACAAdgBlAHIAcwBpAG8AbgA9ACIANAAuADAALgAwAC4AMAAiAD4APABEAEEAVABBAD4APABQAFIATwBUAEUAQwBUAEkATgBGAE8APgA8AEsARQBZAEwARQBOAD4AMQA2ADwALwBLAEUAWQBMAEUATgA+ADwAQQBMAEcASQBEAD4AQQBFAFMAQwBUAFIAPAAvAEEATABHAEkARAA+ADwALwBQAFIATwBUAEUAQwBUAEkATgBGAE8APgA8AEsASQBEAD4AagBlADEAcABvAEoAWABIAHYAMABPAFMATwBZAFQAYgAyAEwATwBIAGkAZwA9AD0APAAvAEsASQBEAD4APABDAEgARQBDAEsAUwBVAE0APgAyAHcAZgBLAEUASwBGAEkAYwAwAFkAPQA8AC8AQwBIAEUAQwBLAFMAVQBNAD4APABMAEEAXwBVAFIATAA+AGgAdAB0AHAAcwA6AC8ALwBjAHAAaQB4AC0AawBlAHkAbwBzAC4AbABpAGMAZQBuAHMAZQBrAGUAeQBzAGUAcgB2AGUAcgAuAGMAbwBtAC8AYwBwAGkAeAAvAHYAMQAuADAALwA8AC8ATABBAF8AVQBSAEwAPgA8AC8ARABBAFQAQQA+ADwALwBXAFIATQBIAEUAQQBEAEUAUgA+AA==</mspr:pro>
        <cenc:pssh>AAACrHBzc2gAAAAAmgTweZhAQoarkuZb4IhflQAAAoyMAgAAAQABAIICPABXAFIATQBIAEUAQQBEAEUAUgAgAHgAbQBsAG4AcwA9ACIAaAB0AHQAcAA6AC8ALwBzAGMAaABlAG0AYQBzAC4AbQBpAGMAcgBvAHMAbwBmAHQALgBjAG8AbQAvAEQAUgBNAC8AMgAwADAANwAvADAAMwAvAFAAbABhAHkAUgBlAGEAZAB5AEgAZQBhAGQAZQByACIAIAB2AGUAcgBzAGkAbwBuAD0AIgA0AC4AMAAuADAALgAwACIAPgA8AEQAQQBUAEEAPgA8AFAAUgBPAFQARQBDAFQASQBOAEYATwA+ADwASwBFAFkATABFAE4APgAxADYAPAAvAEsARQBZAEwARQBOAD4APABBAEwARwBJAEQAPgBBAEUAUwBDAFQAUgA8AC8AQQBMAEcASQBEAD4APAAvAFAAUgBPAFQARQBDAFQASQBOAEYATwA+ADwASwBJAEQAPgBqAGUAMQBwAG8ASgBYAEgAdgAwAE8AUwBPAFkAVABiADIATABPAEgAaQBnAD0APQA8AC8ASwBJAEQAPgA8AEMASABFAEMASwBTAFUATQA+ADIAdwBmAEsARQBLAEYASQBjADAAWQA9ADwALwBDAEgARQBDAEsAUwBVAE0APgA8AEwAQQBfAFUAUgBMAD4AaAB0AHQAcABzADoALwAvAGMAcABpAHgALQBrAGUAeQBvAHMALgBsAGkAYwBlAG4AcwBlAGsAZQB5AHMAZQByAHYAZQByAC4AYwBvAG0ALwBjAHAAaQB4AC8AdgAxAC4AMAAvADwALwBMAEEAXwBVAFIATAA+ADwALwBEAEEAVABBAD4APAAvAFcAUgBNAEgARQBBAEQARQBSAD4A</cenc:pssh>
      </ContentProtection>
      <!-- Widevine -->
      <ContentProtection schemeIdUri="urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed">
        <cenc:pssh>AAAAV3Bzc2gAAAAA7e+LqXnWSs6jyCfc1R0h7QAAADcIARIQoGntjceVQ7+SOYTb2LOHihoLYnV5ZHJta2V5b3MiEKBp7Y3HlUO/kjmE29izh4oqAkhE</cenc:pssh>
      </ContentProtection>
      <SegmentTemplate duration="10001" initialization="$RepresentationID$/init.mp4" media="$RepresentationID$/seg-$Number$.m4s" startNumber="1" timescale="1000"/>
      <Representation bandwidth="1280119" codecs="avc1.64001E" frameRate="25" height="360" id="video/avc1/1" scanType="progressive" width="640"/>
      <Representation bandwidth="2242197" codecs="avc1.64001F" frameRate="25" height="480" id="video/avc1/2" scanType="progressive" width="856"/>
      <Representation bandwidth="3127693" codecs="avc1.64001F" frameRate="25" height="576" id="video/avc1/3" scanType="progressive" width="1024"/>
    </AdaptationSet>
    <!-- Audio -->
    <AdaptationSet mimeType="audio/mp4" segmentAlignment="true" startWithSAP="1">
      <!-- MPEG Common Encryption -->
      <ContentProtection cenc:default_KID="a069ed8d-c795-43bf-9239-84dbd8b3878a" schemeIdUri="urn:mpeg:dash:mp4protection:2011" value="cenc"/>
      <!-- PlayReady -->
      <ContentProtection schemeIdUri="urn:uuid:9a04f079-9840-4286-ab92-e65be0885f95" value="2.0">
        <mspr:pro>jAIAAAEAAQCCAjwAVwBSAE0ASABFAEEARABFAFIAIAB4AG0AbABuAHMAPQAiAGgAdAB0AHAAOgAvAC8AcwBjAGgAZQBtAGEAcwAuAG0AaQBjAHIAbwBzAG8AZgB0AC4AYwBvAG0ALwBEAFIATQAvADIAMAAwADcALwAwADMALwBQAGwAYQB5AFIAZQBhAGQAeQBIAGUAYQBkAGUAcgAiACAAdgBlAHIAcwBpAG8AbgA9ACIANAAuADAALgAwAC4AMAAiAD4APABEAEEAVABBAD4APABQAFIATwBUAEUAQwBUAEkATgBGAE8APgA8AEsARQBZAEwARQBOAD4AMQA2ADwALwBLAEUAWQBMAEUATgA+ADwAQQBMAEcASQBEAD4AQQBFAFMAQwBUAFIAPAAvAEEATABHAEkARAA+ADwALwBQAFIATwBUAEUAQwBUAEkATgBGAE8APgA8AEsASQBEAD4AagBlADEAcABvAEoAWABIAHYAMABPAFMATwBZAFQAYgAyAEwATwBIAGkAZwA9AD0APAAvAEsASQBEAD4APABDAEgARQBDAEsAUwBVAE0APgAyAHcAZgBLAEUASwBGAEkAYwAwAFkAPQA8AC8AQwBIAEUAQwBLAFMAVQBNAD4APABMAEEAXwBVAFIATAA+AGgAdAB0AHAAcwA6AC8ALwBjAHAAaQB4AC0AawBlAHkAbwBzAC4AbABpAGMAZQBuAHMAZQBrAGUAeQBzAGUAcgB2AGUAcgAuAGMAbwBtAC8AYwBwAGkAeAAvAHYAMQAuADAALwA8AC8ATABBAF8AVQBSAEwAPgA8AC8ARABBAFQAQQA+ADwALwBXAFIATQBIAEUAQQBEAEUAUgA+AA==</mspr:pro>
        <cenc:pssh>AAACrHBzc2gAAAAAmgTweZhAQoarkuZb4IhflQAAAoyMAgAAAQABAIICPABXAFIATQBIAEUAQQBEAEUAUgAgAHgAbQBsAG4AcwA9ACIAaAB0AHQAcAA6AC8ALwBzAGMAaABlAG0AYQBzAC4AbQBpAGMAcgBvAHMAbwBmAHQALgBjAG8AbQAvAEQAUgBNAC8AMgAwADAANwAvADAAMwAvAFAAbABhAHkAUgBlAGEAZAB5AEgAZQBhAGQAZQByACIAIAB2AGUAcgBzAGkAbwBuAD0AIgA0AC4AMAAuADAALgAwACIAPgA8AEQAQQBUAEEAPgA8AFAAUgBPAFQARQBDAFQASQBOAEYATwA+ADwASwBFAFkATABFAE4APgAxADYAPAAvAEsARQBZAEwARQBOAD4APABBAEwARwBJAEQAPgBBAEUAUwBDAFQAUgA8AC8AQQBMAEcASQBEAD4APAAvAFAAUgBPAFQARQBDAFQASQBOAEYATwA+ADwASwBJAEQAPgBqAGUAMQBwAG8ASgBYAEgAdgAwAE8AUwBPAFkAVABiADIATABPAEgAaQBnAD0APQA8AC8ASwBJAEQAPgA8AEMASABFAEMASwBTAFUATQA+ADIAdwBmAEsARQBLAEYASQBjADAAWQA9ADwALwBDAEgARQBDAEsAUwBVAE0APgA8AEwAQQBfAFUAUgBMAD4AaAB0AHQAcABzADoALwAvAGMAcABpAHgALQBrAGUAeQBvAHMALgBsAGkAYwBlAG4AcwBlAGsAZQB5AHMAZQByAHYAZQByAC4AYwBvAG0ALwBjAHAAaQB4AC8AdgAxAC4AMAAvADwALwBMAEEAXwBVAFIATAA+ADwALwBEAEEAVABBAD4APAAvAFcAUgBNAEgARQBBAEQARQBSAD4A</cenc:pssh>
      </ContentProtection>
      <!-- Widevine -->
      <ContentProtection schemeIdUri="urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed">
        <cenc:pssh>AAAAV3Bzc2gAAAAA7e+LqXnWSs6jyCfc1R0h7QAAADcIARIQoGntjceVQ7+SOYTb2LOHihoLYnV5ZHJta2V5b3MiEKBp7Y3HlUO/kjmE29izh4oqAkhE</cenc:pssh>
      </ContentProtection>
      <SegmentTemplate duration="10001" initialization="$RepresentationID$/init.mp4" media="$RepresentationID$/seg-$Number$.m4s" startNumber="1" timescale="1000"/>
      <Representation audioSamplingRate="44100" bandwidth="134479" codecs="mp4a.40.2" id="audio/und/mp4a">
        <AudioChannelConfiguration schemeIdUri="urn:mpeg:dash:23003:3:audio_channel_configuration:2011" value="2"/>
      </Representation>
    </AdaptationSet>
  </Period>
```

**Comment by SebiderSushi** [comment]
TL;DR: @cowcrash i think your original issue has been fixed during the last month but as of now tele5.de has implemented DRM protection on part of their videos and seems to enforce it on some of them.

**Comment by cowcrash** [comment]
@SebiderSushi thanx for your efforts

**Comment by SebiderSushi** [comment]
No Problem i encountered this issue myself so i got curious.

**Comment by 89z** [comment]
Issue should be closed, URL is dead
